open-ssid-list.txt

```diff
@@ -1,66 +1,156 @@
+#TELUSDirect
 #WiFi@Changi
-AIRPORT
 .Venice Airport Free WiFi
+A&W Free Wi-Fi
+ADPGUESTWIFI
+AIRPORT
+ASUS
 AirPort Express
 Airport_FreeInternet
 Apple Demo
+Apt3Guest
 Auckland Airport
 BAYMONT
+BELL762-10-guest
```

Comment:
What's BELL762? Is it a specific location?

Reply:
That might have snuck in by mistake, it was an open AP however looking now I believe it was just a personal one. I'll remove it.